assets/css/app.scss

```diff
@@ -1,3 +1,5 @@
+@use 'sass:math';
```

Comment:
`@use` rules must be written before any other rules in a file. I decided to put it here in the root Sass file since it's a generic rule.